verticapy/connection/connect.py

```diff
@@ -125,6 +129,52 @@ def connect(section: str, dsn: Optional[str] = None) -> None:
         prev_conn.close()
     try:
         gb_conn.set_connection(vertica_connection(section, dsn), section, dsn)
+        print("Connected Successfuly!")
+    except OAuthTokenRefreshError as e:
+        error_message = str(e)
+
+        from verticapy.connection.write import new_connection
+
+        # Check if the error message contains the refresh token part
+        if "Failed getting OAuth access token from a refresh token" in error_message:
```

Comment:
vertica-python is raising an 'OAuthTokenRefreshError'
Can/should we check for this instead of checking the specific error message? The error message could conceivably change, but the exception type should not.